lib/containers/container_images.pm

```diff
@@ -129,4 +129,11 @@ sub test_opensuse_based_image {
     assert_script_run("$runtime run --rm refreshed-image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
 }
 
+sub exec_cmd {
+    my ($image, $runtime, $command, $timeout) = @_;
+    $timeout //= 120;
+    assert_script_run("$runtime run --rm $image $command", $timeout);
+    save_screenshot;
```

Comment:
why do we need this?

Reply:
I dont think it is really needed. i used it in my local instance to capture the screenshot after the assert_script_run. But maybe it doe not make sense here. i will remove it

Reply:
Ok. In our tests we use `_QUIET_SCRIPTS_CALLS` as all the output goes to `serial_terminal.txt`

Reply:
removed